screwdriver/screwdriver.go

```diff
@@ -292,6 +300,21 @@ func (a api) BuildFromID(buildID int) (build Build, err error) {
 	return build, nil
 }
 
+// EventFromID fetches and returns a Event object from its ID
+func (a api) EventFromID(eventID int) (event Event, err error) {
+	u, err := a.makeURL(fmt.Sprintf("events/%d", eventID))
+	body, err := a.get(u)
+	if err != nil {
+		return event, err
+	}
+
+	err = json.Unmarshal(body, &event)
```

Comment:
Why bother with returning an `Event` object when you're modifying what's passed in anyways?